resources/language/resource.language.es_es/strings.po

```diff
@@ -292,11 +294,11 @@ msgstr "Vista para los exportados"
 
 msgctxt "#30075"
 msgid "Ask to skip intro and recap"
-msgstr "Preguntar si omitir intro y resumen"
+msgstr "Preguntar si omitir la introducción y resumen"
 
 msgctxt "#30076"
 msgid "Skip intro"
-msgstr "Omitir intro"
+msgstr "Omitir introducción"
 
```

Comment:
DON'T AGREE. The reason is the same as above.

Reply:
DON'T AGREE, I don't like the abbreviations

Reply:
I think that the ideal is to leave it as in the application and the web application. 

https://user-images.githubusercontent.com/24685253/79220492-611f1100-7e54-11ea-8460-e203acdb5a7d.png